Modules/Devices/src/Devices.Application/Identities/Queries/IsIdentityOfUserDeleted/IsIdentityOfUserDeletedQuery.cs

```diff
@@ -0,0 +1,13 @@
+﻿using MediatR;
+
+namespace Backbone.Modules.Devices.Application.Identities.Queries.IsIdentityOfUserDeleted;
+
+public class IsIdentityOfUserDeletedQuery : IRequest<IsIdentityOfUserDeletedResponse>
+{
+    public IsIdentityOfUserDeletedQuery(string username)
+    {
+        Username = username;
+    }
```

Comment:
Well, we don't have a common guideline at the moment. I tend to say that we should go with init properties instead of constructor parameters. But I haven't made the decision yet.
I will change it here anyway.